jobs/furnishings/report-templates/noticeOfDissolutionCommencement.html

```diff
@@ -32,23 +32,28 @@
+          {% endif %}
         {% else %}
           <div class="mt-5"><span class="bold">No Post Restoration Transition Application Filed Since {{ furnishing.lastARDate }} for {{ furnishing.businessIdentifier }}</span></div>
           <div class="mt-4">To file a Post Restoration Transition Application (PRTA) online, log in to your Business Page at &nbsp;&nbsp; <a href="https://www.business.bcregistry.gov.bc.ca/{{ furnishing.businessIdentifier }}" class="break-url">business.bcregistry.gov.bc.ca/<wbr>{{ furnishing.businessIdentifier }}</a> to file any outstanding annual reports listed.</div>
         {% endif %}
         {% if 'NO_AR' in furnishing.furnishingName %}
           {% if 'XPRO' in furnishing.furnishingName %}
-            <div class="mt-5">Under section 422 of the Business Corporation Act (the Act), this letter is to notify you that your extraprovincial company has for two years failed to file the annual reports required under section 51 of the Act. A company must annually, within two months after each anniversary of the date on which the company was recognized, file an annual report with the Registrar.</div>
-            <div class="mt-5">If within one month after the date of this letter, the company fails to file the outstanding annual reports, a notice may be published on the BC Laws website <a href="https://www.bclaws.gov.bc.ca/" class="break-url">www.bclaws.ca</a>. This notice will state that, at any time after the expiration of one month after the date of publication of the notice, the company will be cancelled, unless cause is shown to the contrary; I am satisfied the failure has been or is being remedied; or a copy of the entered court order to the contrary has been filed.</div>
+            <div class="mt-5">Under section 422 of the Business Corporation Act (the Act), this letter is to notify you that your extraprovincial company has for two years failed to file the annual reports required under section 380 of the Act. An extraprovincial company must annually, within two months after each anniversary of the date of its registration, file an annual report with the registrar.</div>
+            <div class="mt-5">If within one month after the date of this letter, the company fails to file the outstanding annual reports, a notice may be published on the BC Laws website <a href="https://www.bclaws.gov.bc.ca/" class="break-url">www.bclaws.ca</a>. This notice will state that, at any time after the expiration of one month after the date of publication of the notice, the company’s registration may be cancelled, unless cause is shown to the contrary; The Registrar satisfied the failure has been or is being remedied; or a copy of the entered court order to the contrary has been filed.</div>
```

Comment:
<img width="1289" alt="image" src="https://github.com/user-attachments/assets/caafb1a6-e9c0-44cb-891f-ba72f6aa53e3">
I noticed that the letter 't' in 'D1 Dissolution Letter - XPros - policy revision' is in lowercase. Perhaps we could follow the template's style for consistency.

Reply:
Yes, I saw it. But, they are uppercase in the other letters ( BC Corp and PRTA ). So, I made it like that.
@dylan644 can you confirm it?

Reply:
well spotted - let's go with consistency please